harvard-university-of-worcester.csl

```diff
@@ -0,0 +1,376 @@
+            <text variable="number" prefix=", episode " suffix="."/>
+          </if>
+        </choose>
+      </if>
+    </choose>
+  </macro>
+  <macro name="single-quote-title">
+    <text variable="title" prefix=" '" suffix="'."/>
+  </macro>
+  <macro name="translator">
+    <text term="translator" form="short" plural="true" text-case="capitalize-first" suffix="."/>
+    <names variable="translator" prefix=" " suffix="."/>
+  </macro>
+  <macro name="original-title">
+    <text variable="original-title" font-style="italic" prefix="[" suffix="]"/>
+  </macro>
+  <macro name="genre">
+    <text variable="genre"/>
+  </macro>
+  <macro name="volume-number">
+    <number variable="volume"/>
+  </macro>
+  <macro name="issue-number">
+    <number variable="issue"/>
+  </macro>
+  <macro name="event-name">
+    <text variable="event"/>
+  </macro>
+  <macro name="event-place">
+    <text variable="event-place"/>
+  </macro>
+  <macro name="event-date">
+    <date form="text" variable="event-date">
+      <date-part name="day"/>
+      <date-part name="month"/>
+    </date>
+  </macro>
+  <macro name="broadcast-details">
+    <choose>
+      <if type="broadcast interview" match="any">
+        <text variable="publisher" suffix=","/>
+        <date form="text" variable="issued" prefix=" ">
+          <date-part name="day"/>
+          <date-part name="month"/>
+        </date>
+      </if>
+    </choose>
+  </macro>
+  <macro name="scale">
+    <text variable="scale"/>
+  </macro>
+  <macro name="director">
+    <text term="director" prefix=":"/>
+    <names variable="director"/>
+  </macro>
+  <macro name="graph-or-figure">
+    <choose>
+      <if type="graphic" match="any">
+        <text value="graph"/>
+      </if>
+      <else-if type="figure" match="any">
+        <text value="fig"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="day-month">
+    <choose>
+      <if type="webpage interview personal_communication post-weblog" match="any">
+        <date form="text" date-parts="year-month-day" variable="issued">
+          <date-part name="day"/>
+          <date-part name="month"/>
+        </date>
+      </if>
+    </choose>
+  </macro>
+  <macro name="interviewer">
+    <text value="Interviewed by "/>
+    <names variable="interviewer"/>
+    <text value=" for "/>
+  </macro>
+  <macro name="receipient">
+    <names variable="recipient"/>
+  </macro>
+  <citation collapse="year" disambiguate-add-year-suffix="true" et-al-use-first="3" et-al-min="4">
+    <layout suffix=")" delimiter="; " prefix="(">
+      <group delimiter=", ">
+        <text macro="author-short"/>
+        <text macro="year-date-citation"/>
+        <group>
+          <label form="short" variable="locator"/>
+          <text variable="locator"/>
+        </group>
+      </group>
+    </layout>
+  </citation>
+  <bibliography et-al-use-first="4" et-al-min="5" hanging-indent="true">
+    <sort>
+      <key macro="author"/>
```

Comment:
It is a remit for this university to do things differently simply because it can. The guide I am following explicitly states it's ordered by author/editor A-Z so...there we go.